kvirt/kvm/__init__.py

```diff
@@ -159,6 +159,21 @@ def disk_exists(self, pool, name):
         except:
             return False
 
+    def disk_path(self, pool, name):
```

Comment:
cant we use disk_exists instead ?

Reply:
As suggested on IRC, let disk_path function remain in the kvm package (could possibly be added to other providers later).